pkgs/applications/audio/spotify/darwin.nix

```diff
@@ -0,0 +1,28 @@
+{ fetchurl, stdenv, lib, undmg }:
+
+stdenv.mkDerivation rec {
+  version = "1.0.42.151.g19de0aa6";
+  name = "spotify-mac";
+
+  src = fetchurl {
+    url = "http://download.spotify.com/Spotify.dmg";
```

Comment:
I have pulled older versions from here: http://filehippo.com/mac/download_spotify_for_mac/
But I am not sure whether we want to trust a third-party site...